rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -411,6 +411,7 @@ maas_apt_packages:
 maas_pip_packages:
   - rackspace-monitoring-cli
   - ipaddr
+  - swift
```

Comment:
So I see it's actually calling a number of swift Python scripts e.g. swift-recon. In that case the plugin can be adjusted to accept an additional optional argument specifying the venv's bin directory.